python/packages/autogen-core/docs/src/user-guide/agentchat-user-guide/tutorial/agents.ipynb

```diff
@@ -0,0 +1,328 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "# Agents\n",
```

Comment:
Need a WIP warning here to indicate the current status of the API.

Reply:
Added